Automattic/wp-calypso
# Remove extensions and update WooCommerce imports

Removes the `extensions` directory and moves `woocommerce` to a classic section. After the recent changes that move WordPress.com towards a mix of wp-admin and Calypso screens, it's extremely unlikely that the Calypso Extensions project will ever continue in the current form. It's time to remove the `extensions` code, because it requires continued maintenance.

- removes the `hello-dolly`, `wp-super-cache`, and `zoninator` plugins completely
- moves `text-control` and `woocommerce-connect-cart-header` to `client/components` as they are used only by Login and Signup.
- removes the `build-tools/webpack/extensions` code that traversed the `extensions/**/package.json` files and added sections. Now, all sections are hardcoded in one big array.
- updates imports of `woocommerce/*` modules to relative paths as the `woocommerce` webpack alias no longer exists.
- removes the `ExtensionRedirect` component as it's no longer used.
- removes UI in `/plugins` that links to plugin's extension settings if present

**How to test:**
Verify all the `/store` routes on a site that has WooCommerce installed. `/store/:site` should show a simple Dashboard. In the `/stats` section, you should see a "Store" tab that links to Store stats.
